libs.glassfish_logging/external/commons-logging-1.0.4-license.txt

```diff
@@ -0,0 +1,206 @@
+Name: Commons Logging
```

Comment:
The name of the file need to be `logging-api-1.0.4-license.txt`, as the downloaded artifact gets the name `logging-api-1.0.4.jar`. The prefix of the license text needs to either match the artifact name or the `Files` header of the license file must explicitly reference the files.